### PR TITLE
 📝 Improve the pwa function documentation

### PR DESCRIPTION
**Pull request by JunzhouLiu**
Chinese and English versions

### CODE REVIEW

File: docs/additional-package-support.md [hunk 1]
```diff
@@ -5,15 +5,40 @@
...
+  stripPrefix: public
+  verbose: true
+  runtimeCaching:
+    # CDNs - should be cacheFirst, since they should be used specific versions so should not change
+    - urlPattern: /* # If you need to load CDN resources, please configure this option. If not, you can configure it.
+      handler: cacheFirst
+      options:
+        origin: your_websie_url # can be replaced with your url
+```
+For more details, please see the official documentation for [hexo-offline] (https://github.com/JLHwung/hexo-offline)
```
Comment (Molunerfinn) [inline docs/additional-package-support.md]: image
There is an extra space here

File: docs/additional-package-support.md [hunk 2]
```diff
@@ -28,10 +53,65 @@ pwa:
...
+```
+
+You can also quickly create `manifest.json` via [Web App Manifest] (https://app-manifest.firebaseapp.com/). (Web App Manifest requires at least one 512*512 pixel icon)
+
+6. You can check if the PWA configuration is in effect and the configuration is correct through the `Chrome` plugin `Lighthouse`.
+    - Open the blog page
+    - Launch the `Lighthouse` plugin (the `Lighthouse` plugin requires at least one 512*512 pixel icon)
 
-You can create a `manifest.json` fastly by visiting [Web App Manifest](https://app-manifest.firebaseapp.com/)
+For more on PWA (Progressive Enhanced Web Applications), see [Google Tools for Web Developers] (https://developers.google.com/web/tools/lighthouse/audits/address-bar)
```
Comment (Molunerfinn) [inline docs/additional-package-support.md]: image

There is an extra space here

Comment (JunzhouLiu) [reply docs/additional-package-support.md]: ok.

Comment (Molunerfinn) [reply docs/additional-package-support.md]: Still an extra space here?

Comment (Molunerfinn) [reply docs/additional-package-support.md]: Chinese version is OK while English version is error

Comment (JunzhouLiu) [reply docs/additional-package-support.md]: > Chinese version is OK while English version is error

Sorry, I executed git add without saving changes in vscode. I just found this stupid error and wondered why the change didn't take effect.

Comment (JunzhouLiu) [reply docs/additional-package-support.md]: @Molunerfinn Now I fixed these errors, please check

**Comment by Molunerfinn** [comment]
Thanks a lot!

**Pull request merged**